common/buildcraft/lib/BCLibConfig.java

```diff
@@ -139,9 +145,8 @@ public EnumFacing changeFacing(EnumFacing dir) {
         /** {@link TileEntity}'s that implement the {@link IChunkLoadingTile} interface will be loaded, provided they
          * DON'T return null. */
         SELF_TILES,
-    }
 
-    static {
-        configChangeListeners.add(LocaleUtil::onConfigChanged);
+        /** All {@link TileEntity}'s in the world. */
+        ALL_TILES;
```

Comment:
i still think this should never even be an option, the performance impact would be horrific and you will easely even hit the max number of chunks that can be chunkloaded in the forge config on servers and with the only option to unload parts being removing BC TE's i dont see this ever ending well